scipy/integrate/meson.build

```diff
@@ -86,6 +86,7 @@ py3.extension_module('_quadpack',
     fortran_args: fortran_ignore_warnings,
     include_directories: [inc_np, '../_lib/src'],
     link_with: [quadpack_lib, mach_lib],
+    link_language: 'fortran',
```

Comment:
Hmm, this doesn't seem desirable. I think it'd be preferable to find out the root cause and fix it in Meson.

Reply:
The problem is a different one: the sources are misspecified here. This has:
```
['__quadpack.h', '_quadpackmodule.c'] + quadpack_src + mach_src,
```
but `setup.py` has:
```
    config.add_extension('_quadpack',
                         sources=['_quadpackmodule.c'],
                         depends=(['__quadpack.h']
                                  + quadpack_src + mach_src),
```
the `depends=` is unnecessary in Meson, since that dependency tracking is automatic.

Reply:
This was still wrong, addressing in gh-55